specification/servicenetworking/ServiceNetworking.Management/tspconfig.yaml

```diff
@@ -0,0 +1,9 @@
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    arm-types-dir: ../../../../../common-types/resource-management
```

Comment:
`arm-types-dir` should be removed, since the default value should now work in the repo.